Tools/src/Bundle_fi.properties

```diff
@@ -0,0 +1,57 @@
+array.dialog.name=Array
+array.dialog.title=Create Array of Selected Objects:
+array.type.linear=Linear
+array.type.curve=From curve
+array.copies.number=Number of Copies
+array.tool.name=Array...
+array.xstep.label=Step X:
+array.ystep.label=Step Y:
+array.zstep.label=Step Z:
+array.xstep.size=Times X Size
+array.ystep.size=Times Y Size
+array.zstep.size=Times Z Size
+array.step.size=Step Size:
+array.tool.message=You must select one or more objects to create an array from.
+array.follow.curve=Orientation Follows Curve
+array.use.original.position=Use Original Position
+array.use.original.orientation=Use Original Orientation
+array.include.children=Include Children
+array.group=Group
+array.skip.first=Skip First Copy
+array.copies.live=Live Duplicates
+tube.tool.message=You must select a single curve from which to make a tube.
+tube.dialog.name=Tube
+tube.width.label=Tube Width
+tube.cap.ends.label=Cap Ends
+tube.cap.end.open=Open Ends
+tube.cap.end.flat=Flat Ends
+skin.dialog.title=Skin
+button.skin.dialog.move.up=Move Up
+button.skin.dialog.move.down=Move Down
+skin.dialog.reverse=Reverse Direction
+skin.tool.message.curves=You must select two or more curves to create a skin across.
+skin.tool.curves.same.points=All the curves must have the same number of points.
+skin.tool.curve.same.close=You cannot create a skin between a closed curve and an open one.
+skin.tool.curve.same.type=You cannot create a skin between an interpolating curve and an approximating one.
+lathe.tool.mesage=You must select a single curve to lathe.
+lathe.dialog.name=Lathe
+lathe.axis.select.label=Select Lathe Axis:
+lathe.rotation.angle.label=Total Rotation Angle:
+lathe.radius.label=Radius:
+lathe.segments.label=Segments:
+lathe.rotate.endpoints=Line through endpoints
+lathe.rotate.xaxis=X axis
+lathe.rotate.yaxis=Y axis
+lathe.rotate.zaxis=Z axis
+extrude.tool.messsage=You must select a curve or open surface to extrude and, optionally, a curve along which to extrude it.
+extrude.dialog.name=Extrude
+extrude.target.label=Object to Extrude:
+extrude.direction.label=Extrude Direction:
+extrude.xaxis=X
+extrude.yaxis=Y
+extrude.zaxis=Z
+extrude.segments.label=Segments:
+extrude.twist.value.label=Twist (degrees):
+extrude.surface.accuracy.label=Surface Accuracy:
+extrude.follws.curve.label=Orientation Follows Curve
+csg.tool.message=You must select two objects for boolean modelling, at least one of which must be solid.
```

Comment:
Why is there a Finnish properties file with all English localizations?

Reply:
Okay, I see partial translations. Untranslated strings should not be present at all, though.

Reply:
Removed